pkgs/os-specific/linux/kernel/linux-testing.nix

```diff
@@ -3,15 +3,15 @@
 with lib;
 
 buildLinux (args // rec {
-  version = "5.12-rc6";
+  version = "5.13-rc6";
   extraMeta.branch = "5.12";
 
   # modDirVersion needs to be x.y.z, will always add .0
   modDirVersion = if (modDirVersionArg == null) then builtins.replaceStrings ["-"] [".0-"] version else modDirVersionArg;
 
   src = fetchurl {
     url = "https://git.kernel.org/torvalds/t/linux-${version}.tar.gz";
-    sha256 = "0w0zk2byimdbcvn8myqaq0ab6lyd43493fnkv9a1407dimpxb03d";
+    sha256 = "sha256-PunFd6tOsmrsPItp2QX4TEVxHnvvi1BMSwWio/DTlMU=";
```

Comment:
Not sure, if we are allowed to use prefixed hashes in `nixpkgs` yet. @SuperSandro2000 might know.

Reply:
I think it's OK:
```
$ grep -ri "sha256-" | wc -l
3503
```

Reply:
Greping through the code is not a valid way to determine if we should do something or not. If we would use that method all the time we would never get rid of old and deprecated patterns.

Using prefixed hashes is allowed and I have no problem with it.